src/constants.ts

```diff
@@ -30,21 +30,56 @@ export const AuthType = {
   ApiKey: 'cdf-api-key',
 };
 
-export const DateFields = ['lastUpdatedTime', 'createdTime', 'startTime', 'endTime'];
-
+export const DateFields = [
+  'lastUpdatedTime',
+  'createdTime',
+  'startTime',
+  'endTime',
+  'lastSeen',
+  'lastFailure',
+  'lastSuccess',
+];
+const commonFields = ['id', 'externalId'];
 export const EventFields = [
-  'id',
-  'externalId',
+  ...commonFields,
   'type',
   'subtype',
-  'dataSetId',
   'assetIds',
   'source',
   'sourceId',
   'metadata',
+  'dataSetId',
+  ...DateFields,
+];
+const createFields = (parent, field) => `${parent}-${field}`;
+const metaFields = [
+  'Dockerfile',
+  'Extractor config',
+  'branch',
+  'environment',
+  'extractor',
+  'site',
+  'source',
+].map((_) => createFields('metadata', _));
```

Comment:
are those some customer specific fields, or is a part of common practice for the extraction pipelines?

Reply:
they are the object as value from the Resource response which will be supported under the table as values by 'sourceKey-key' , this is the declaration of tableColumName for the key metadata from the EP response Object, to be present when adding new column to table.